docs/api/data-views/create.asciidoc

```diff
@@ -44,64 +41,214 @@ the data view is stored. The default is `false`.
+          "machine.os.keyword": { "agg": "terms" }
+        },
+        "date_histogram": {
+          "@timestamp": {
+            "agg": "date_histogram",
+            "fixed_interval": "20m",
+            "delay": "10m",
+            "time_zone": "UTC"
+          }
+        },
+        "avg": {
+          "memory": { "agg": "avg" },
+          "bytes": { "agg": "avg" }
+        },
+        "max": { "memory": { "agg": "max" } },
+        "min": { "memory": { "agg": "min" } },
+        "sum": { "memory": { "agg": "sum" } },
+        "value_count": { "memory": { "agg": "value_count" } },
+        "histogram": {
+          "machine.ram": {
+            "agg": "histogram",
+            "interval": 5
+          }
+        }
+      }
+    }
+  }
+}
+--------------------------------------------------
+// KIBANA
 
-The API returns the data view object:
+The API returns the {data-source} object:
 
 [source,sh]
 --------------------------------------------------
 {
     "data_view": {...}
 }
 --------------------------------------------------
+
+
+[[data-views-api-properties]]
+
+==== Properties of the `data_view` object:
+
+`title`::
+(Optional, string) Comma-separated list of data streams, indices, and aliases that you want to search. Supports wildcards
+(`*`).
+
+`name`::
+(Optional, string) The {data-source} name.
+
+`id`::
+(Optional, string) Saved object ID.
+
+`type`::
+(Optional, string) When set to `rollup`, identifies the rollup {data-sources}.
+
+`typeMeta`::
+(Optional, object) When you use rollup indices, contains the field list for the rollup
+{data-source} API endpoints.
++
+.Properties of the typeMeta objects:
+[%collapsible%open]
+=====
+`aggs`:::
+(Required, object) A map of rollup restrictions by aggregation type and field name.
+
+`params`:::
+(Required, object) Properties for retrieving rollup fields.
+=====
+
+`timeFieldName`::
+(Optional, string) Timestamp field name, which you use for time-based {data-sources}.
+
+`sourceFilters`::
+(Optional, string[]) Array of field names you want to filter out in <<discover, **Discover**>>.
+
+`fieldAttrs`::
+(Optional, object) Map of field attributes by field name.
++
+.Properties of the fieldAttrs[fieldName] objects:
+[%collapsible%open]
+=====
+`customLabel`:::
+(Optional, string) Custom label for the field.
+
+`count`:::
+(Optional, number) Popularity count for the field.
+=====
+
+`runtimeFieldMap`::
+(Optional, object) Map of runtime field definitions by field name.
++
+.Properties of the runtimeFieldMap[fieldName] objects:
+[%collapsible%open]
+=====
+`type`:::
+(Required, string) Mapping type of the runtime field. For more information, check {ref}/mapping-types.html[Field data types].
+
+`script.source`:::
+(Required, string) Script of the runtime field.
+=====
+
+`fieldFormats`::
+(Optional, object) Map of field formats by field name.
+
+`allowNoIndex`::
+(Optional, boolean) Allows the {data-source} saved object to exist before the data is available.
```

Comment:
@mattkime looks like this field is ignored in the REST API. Is it a bug, or should we remove this?

Reply:
I just took a look at this code and it appears to be used as expected. Feel free to show me otherwise.